.github/workflows/build.yml

```
@@ -0,0 +1,34 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Check whether minirust succeeds to build.
```

```suggestion
    name: specr build
```
The current name doesn't even fit the column in the actions view^^